app/components/non_bordered_cards_component/non_bordered_cards_component.html.erb

```diff
@@ -12,7 +12,7 @@
       <% end %>
       <p class="govuk-body">
         <% if card[:link][:link_url].present? %>
-          <%= link_to card[:link][:link_title], card[:link][:link_url], class: [{ 'ncce-link': card[:link][:button].blank?, "govuk-button button button--full-width": card[:link][:button].present? }], data: {
+          <%= link_to card[:link][:link_title], card[:link][:link_url], method: (card[:link][:link_method].presence || :get) ,class: [{ 'ncce-link': card[:link][:button].blank?, "govuk-button button button--full-width": card[:link][:button].present? }], data: {
```

Comment:
This looks like what once was a good idea, but now is just too much. Can we define a method in the view component which breaks down these conditionals into more than just one expression?